### DISCUSSION TITLE
EOS Scam Sites

### DISCUSSION THREAD

**Discussion by dguruwu** [StevenBlack/hosts#1960] [Ideas]
I am new to the EOS blockchain community and there are a number of known scam sites I wanted to suggest to this list.

https://help.eossupport.io/en/articles/5715406-list-of-known-scam-websites-url

eosuniswap.io
eos-uniswap.com  (this is modified from original for a reason)
bloks.io/account/uniswap.co (don't want to block entire domain but just this subdirectory but after testing can only block entire domain)
voice.com-register.online
voice.com-register.top
eos-voice.live
rebrand.ly/voice-social-56d70 (I assume the same for this link as above bloks.io link but not entirely certain)
eos-voice.best
eosshiba.com
walletinconnect.com
walletprotector.io
dappmodule.com
bip39converttool.io

I don't know how to use github really and am not a developer or coder so please forgive me in advance if I am doing it wrong.

Was about to test to see if you can exclude directories or subdirectories only or if hosts file works only by domain.

Was going to have these all point to 0.0.0.0 but went looking for another trusted source of scam sites and found this repo, which I am still trying to figure out how to use.

**Comment by dguruwu** [comment]
Does this repo even concern itself with scams?  I don't see that as a category.

**Comment by SlimTux** [comment]
> I am new to the EOS blockchain community and there are a number of known scam sites I wanted to suggest to this list.
> 
> https://help.eossupport.io/en/articles/5715406-list-of-known-scam-websites-url
> 
> eosuniswap.io eos-uniswap.com (this is modified from original for a reason) bloks.io/account/uniswap.co (don't want to block entire domain but just this subdirectory but after testing can only block entire domain) voice.com-register.online voice.com-register.top eos-voice.live rebrand.ly/voice-social-56d70 (I assume the same for this link as above bloks.io link but not entirely certain) eos-voice.best eosshiba.com walletinconnect.com walletprotector.io dappmodule.com bip39converttool.io
> 
> I don't know how to use github really and am not a developer or coder so please forgive me in advance if I am doing it wrong.
> 
> Was about to test to see if you can exclude directories or subdirectories only or if hosts file works only by domain.
> 
> Was going to have these all point to 0.0.0.0 but went looking for another trusted source of scam sites and found this repo, which I am still trying to figure out how to use.

Hey ( just to clarify, i am not related to the project nor have authority over it) if you want to add stuff to the blocklist you can fork the project and do your own version of the projec, adding or removing stuff as you wish. If you want to see an example how does this works you can enter my profile and see my fork (since i added stuff too), or you can watch a  tutorial that i made ( i will not share the link here, because idk if it is allowed) just hit me up on github.
Best regards, SlimTux.
